src/content/development/contribution-guide/_index.en.md

```diff
@@ -62,7 +62,34 @@ To be expanded
 
 #### Feature Design
 
-To be expanded
+Project members are expected to work on the design for any [epic](#epics) features assigned to them.
+During this sprint, project members will update their respective epics with any work identified during the design phase.
+Project members are encouraged to perform proof of concept investigations in order to validate the design and clarify specific work items.
+
+In case additional work items are identified during the design, they should be opened as [tasks](#tasks) and tracked under the respective epic.
+Such tasks are expected to follow the sizing guidelines from the [Pre-Planning](#pre-planning) stage.
+Specifically, tasks that are themselves epics due to their size should be identified and treated as such.
+
+Design proposals for epics should be submitted as [pull requests to the enhancements repository], detailing the proposed design, any
+alternatives, and any changes necessary to the Submariner projects and APIs.
+
+The [pull requests to the enhancements repository] will be reviewed during the sprint, discussing any necessary changes or reservations.
+Any pull request will need approval from at least 50% of the [code owners of the enhancements repository].
+When the pull request is reviewed and merged, work on the epic can begin.
```

Comment:
Maybe add that there is no need to wait to the end of the design sprint, and work on an epic can begin once the enhance PR is merged

Reply:
This is exactly what this line is saying, do you have a better way to say it?

It seems to me "work on the epic can begin" is pretty straightforward but I might be wrong

Reply:
I agree that the current version is clear, but maybe something like:

> As soon as the enhancement pull request is reviewed and merged, work on the epic can begin.

Would emphasize @maayanf24's point that there's no need to wait until the end of the sprint.